.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,40 +13,37 @@ Read and tick the following checkbox after you have created the issue or place a
 
 * [ ] I have read the [CONTRIBUTING.md](https://github.com/nextcloud/news/blob/master/CONTRIBUTING.md) and followed the provided tips
 * [ ] I accept that the issue will be closed without comment if I do not check [here](https://github.com/nextcloud/news/blob/master/CONTRIBUTING.md#before-reporting-bugs)
-* [ ] I accept that the issue will be closed without comment if I don not fill out all items in the issue template.
+* [ ] I accept that the issue will be closed without comment if I do not fill out all items in the issue template.
 
 
 ## Explain the Problem
 What problem did you encounter?
 
 ## Steps to Reproduce
 Explain what you did to encounter the issue
-1.
-2.
-3.
+1. Click
+2. Drag
+3. ??
+4. Profit!
 
 ## System Information
 * News app version:
 * Nextcloud version:
-* Cron type: 
+* Cron type: (system cron/python updater/...)
 * PHP version:
 * Database and version:
 * Browser and version:
-* Distribution and version:
+* OS and version:
 
-## Contents of nextcloud/data/nextcloud.log
+<details>
+<summary>Contents of <em>nextcloud/data/nextcloud.log</em></summary>
 
-```json
+    Paste output here, don't forget the 4 space indentation.
```

Comment:
I wouldn't recommend using the indentation instead of 3 backtick (```) blocks, since sometimes there are multiple lines or a preformatted message which is a pain to indent it manually

Reply:
That one doesn't work with the hiding though. (and you can select and tab to indent in github's UI)

Reply:
It does but you have to add at least one newline between summary and the backticks (edit this comment to see).

<details>
<summary>Contents of <em>nextcloud/data/nextcloud.log</em></summary>

```
 Paste output here, don't forget the 4 space indentation.
Some preformatted data
with newlines and    spaces
```
</details>